test/Model/CategoryEnumTest.php

```diff
@@ -35,7 +35,7 @@
  * CategoryEnumTest Class Doc Comment
  *
  * @category    Class
- * @description CategoryEnum
+ * @description * &#x60;hris&#x60; - hris * &#x60;ats&#x60; - ats * &#x60;accounting&#x60; - accounting * &#x60;ticketing&#x60; - ticketing * &#x60;crm&#x60; - crm * &#x60;mktg&#x60; - mktg * &#x60;filestorage&#x60; - filestorage
```

Comment:
are changes like these expected?

Reply:
frankly idk, but it seems like OpenAPI just changed the way it lists out Enums between the last time we updated this and now. The important thing to check more so is that functionally the SDK doesn't have any weird changes, things in comments and stuff we can probably let go